src/main/java/com/serhiiklymchuk/gmailpet/controller/LoginController.java

```diff
@@ -0,0 +1,24 @@
+package com.serhiiklymchuk.gmailpet.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
```

Comment:
avoid * imports.
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports